### DISCUSSION TITLE
Looking for Maintainers

### DISCUSSION THREAD

**Discussion by AlexvZyl** [AlexvZyl/nordic.nvim#96] [Announcements]
Hi everyone,

I am currently looking for maintainers.  I have been using this theme for a year now and, as one does, want to start using other themes as well.  This means that I will spend less time maintaining and fixing stuff, but I most definitely will still do it.

If anyone is keen to help maintain, let me know in this discussion.

It is not that hard to maintain, colorschemes are relatively straightforward.  However, treesitter updates frequently and can be a hassle.

I would still like to review all PRs, but I won't be too pedantic ;)

**Comment by 5-pebbles** [comment]
I am very interested, however I am not all that good at Lua, and I have no experience maintaining a public repo.

If there is anything that needs to be done, though, I would be happy to work on it.

**Reply by AlexvZyl** [reply]
Awesome.  

Don't worry about your experience, it's not a problem.

I am still on vacation until the 7th of Jan, then I will be a bit more active and review your PR etc. :)

**Comment by 5-pebbles** [comment]
Great, enjoy your vacation!

**Comment by 5-pebbles** [comment]
I am now fairly confident I can maintain this just fine, I have gone over every line in the codebase more than once.

I have been using my own fork since January 3 and I think it's currently better than the upstream repo.

If the current dev branch is merged, the two will start to diverge because I need the config to allow reloading.

&nbsp;

I would like to create a PR for it but my last PR has been waiting for 55 days.

I don't want to be rude, and I am sure you are busy, but are you interested in merging the two.

&nbsp;

If so, I will fix issues, and create PRs for them. If not, I will continue to add comments on issues I know the answer to, but it does not make sense for me to keep up with two codebases.

**Reply by AlexvZyl** [reply]
That's great news @5-pebbles.  Let me sort out the stuff that is currently on my plate (the stuff concerning the state) and then we can discuss going forward!

**Reply by AlexvZyl** [reply]
Again, apologies for leaving stuff hanging.

**Reply by 5-pebbles** [reply]
I have state handled in https://github.com/5-pebbles/nordic.nvim/tree/refactor

But it builds on top of #101
